documentation/tut/toy-example-from-scratch.rst

```diff
@@ -171,7 +171,14 @@ Yes, that is quite a large battery :)
 
 .. note:: Obviously, you can use the ``flexmeasures`` command to create your own, custom account and assets. See :ref:`cli`. And to create, edit or read asset data via the API, see :ref:`v3_0`.
 
-We can also look at the battery asset in the UI of FlexMeasures (in Docker, the FlexMeasures web server already runs, on your PC you can start it with ``flexmeasures run``).
+We can also look at the battery asset in the UI of FlexMeasures (in Docker, the FlexMeasures web server already runs, on your PC you can start it with ``flexmeasures run``), but before this we need to add two environment variables.
```

Comment:
1. This does not apply in the Docker case, as there we add these env variables. I would actually add this further up, in the setup of "On your PC", instead of here.
2. To keep this tutorial simple, we could also use `export` to set these env variables, like we already do with `SQLALCHEMY_DATABASE_URI`.